src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -20,7 +21,8 @@ interface Props {
 const ContentInsightsSummary = ({ error, graphData, isLoading, previousPeriodCount, onClick, totalCount }: Props) => {
     const renderContentAnalyticsSummary = () => {
         if (error) {
-            return <ContentAnalyticsErrorState />;
+            const isPermissionError = error.status === 403;
+            return <ContentAnalyticsErrorState isPermissionError={isPermissionError} />;
```

Comment:
Could we pass the error directly into `ContentAnalyticsErrorState` component and let it figure out what to show?

Reply:
Yeah we can do that

Reply:
> The Permission error icon looks pretty small, is that the desired size?

Mocks seem to have it ranging from 42 - 72 depending on where its at, currently default is 56